docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md

```diff
@@ -0,0 +1,122 @@
+---
+description: Provides guidance on how to view the auto-update and repair settings of a Windows App.
+external help file: Microsoft.Windows.Appx.PackageManager.Commands.dll-help.xml
+Module Name: Appx
+ms.date: 06/07/2021
+online version: https://docs.microsoft.com/powershell/module/appx/Get-AppxPackageAutoUpdateSettings?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+---
+
+# Get-AppxPackageAutoUpdateSettings
+
+## SYNOPSIS
+Available in the Windows Insider Preview Builds of windows 10, is the `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet. The `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides visibility to the settings configured on a Windows 10 client device for a particular Windows App. 
+
+## SYNTAX
+
+```
+Get-AppxPackageAutoUpdateSettings [[-PackageFullName] <String>] [-ShowUpdateAvailability] [-AllUsers]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+The `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet will return the settings configured for a specific or all installed Windows Apps in relation to Auto Update and Repair
+
+## EXAMPLES
+
+### Example 1: Get All App Package Auto Update Settings
+```
+PS C:\> Get-AppxPackageAutoUpdateSettings
+```
+
+This will return the Auto Update and Repair settings for all configured, and installed Windows Apps on the device, and registered to the user.
+
+### Example 2: Get App App Package Auto Update Settings for all users
+```
+PS C:\> Get-AppxPackageAutoUpdateSettings -AllUsers
+```
+
+This will return the Auto Update and Repair settings for all configured and isntalled Windows Apps that have been registered for all users.
```

Comment:
```suggestion
This will return the Auto Update and Repair settings for all configured and installed Windows Apps that have been registered for all users.
```